doc/index.md

```diff
@@ -627,6 +636,19 @@ Params:
 Returns: a promise that fullfils once the default
 settings and active sessions have been updated (if specified)
 
+#### `updateDefaultMediaSettings()`
+Update the default audio volume as well as the audio volume on all active applicable sessions.
```

Comment:
NIT: maybe add the same info as the code comment on the function: 

> Updates the audio volume for all active applicable sessions
_as well as the default volume for future sessions_